web/src/api/views.py

```diff
@@ -89,6 +89,17 @@ def get_block_by_height(request, height):
     register_task(task_id, tasks.get_block_by_height, height)
     return task_result_response(task_id)
 
+@api_view(['GET'])
+def get_blocks_range(request, start_height, end_height):
```

Comment:
to już się robi w inny sposób, pozbyłem się tej redundancji, wystarczy że dodasz to do api/urls.py